src/main/java/com/conveyal/gtfs/loader/JdbcGtfsLoader.java

```diff
@@ -544,7 +552,7 @@ public void setValueForField(Table table, int fieldIndex, int lineNumber, Field
      */
     private void setFieldToNull(boolean postgresText, String[] transformedStrings, int fieldIndex, Field field) {
         if (postgresText) transformedStrings[fieldIndex + 1] = POSTGRES_NULL_TEXT;
-        // Adjust parameter index by two: indexes are one-based and the first one is the CSV line number.
+            // Adjust parameter index by two: indexes are one-based and the first one is the CSV line number.
         else try {
             // LOG.info("setting {} index to null", fieldIndex + 2);
             field.setNull(insertStatement, fieldIndex + 2);
```

Comment:
It'd probably look nice to format this section with brackets around the if/else blocks.

Reply:
Updated.